docs/guide/built-in-functions.md

```diff
@@ -12,17 +12,24 @@ but without the constraints of the spreadsheet UI.
 
 The latest version of HyperFormula has an extensive collection of **{{ $page.functionsCount }}** functions grouped into 11 categories:
 
-* [Date and time](#date-and-time)
-* [Engineering](#engineering)
-* [Information](#information)
-* [Financial](#financial)
-* [Logical](#logical)
-* [Lookup and reference](#lookup-and-reference)
-* [Math and trigonometry](#math-and-trigonometry)
-* [Matrix functions](#matrix-functions)
-* [Operator](#operator)
-* [Statistical](#statistical)
-* [Text](#text)
+- [Built-in functions](#built-in-functions)
+  - [Overview](#overview)
+  - [Language packs](#language-packs)
+  - [Custom functions](#custom-functions)
+  - [Demo](#demo)
+  - [List of available functions](#list-of-available-functions)
+    - [Array manipulation](#array-manipulation)
+    - [Date and time](#date-and-time)
+    - [Engineering](#engineering)
+    - [Information](#information)
+    - [Financial](#financial)
+    - [Logical](#logical)
+    - [Lookup and reference](#lookup-and-reference)
+    - [Math and trigonometry](#math-and-trigonometry)
+    - [Matrix functions](#matrix-functions)
+    - [Operator](#operator)
+    - [Statistical](#statistical)
+    - [Text](#text)
```

Comment:
Something went wrong here? The line above says "unctions grouped into 11 categories:" but then the categories are nested here somehow in full table of contents?

Reply:
@wojciechczerniak - sorry, it was my Markdown VS Code extension going rogue. Fixed now: 6966d9e